chapters/install/ide-speedup.md

```diff
@@ -0,0 +1,90 @@
+# Speed-up Jina App development in PyCharm and VSCode
+
+Jina App development can be time consuming if you donot have the IDE setup correctly. This chapter describes how to speed up your developer experience
+in Pycharm and VS Code IDE while using Jina. In particular we would focus on how to enable Intellisense for Jina YAML configs. Intellisense can provide code completion, showing number and arguments list, validating your YAML config.
```

Comment:
'we would focus on'  <- past tense. 
We use the [present tense ](https://github.com/jina-ai/docs/blob/master/page_templates/style_guide.md)